### PR TITLE
Add 1.18.7 releasenote

### PR DESCRIPTION
**Pull request by hanxiaop**
## Reviewers

- [ ] Ambient
- [x] Docs
- [ ] Installation
- [ ] Networking
- [ ] Performance and Scalability
- [ ] Extensions and Telemetry
- [ ] Security
- [ ] Test and Release
- [ ] User Experience
- [ ] Developer Infrastructure
- [ ] Localization/Translation

### CODE REVIEW

**Comment by hanxiaop** [comment]
The release date has not been determined yet, but it will be soon. I'll update the date once the build is ready.

File: content/en/news/releases/1.18.x/announcing-1.18.7/index.md [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+
+- **Fixed** an issue where `istioctl proxy-config ecds` didn't display all `EcdsConfigDump`.
+
+- **Fixed** an issue where new endpoints may not be sent to proxies.
+  ([Issue #48373](https://github.com/istio/istio/issues/48373))
+
+- **Fixed** an issue where installing with Stackdriver and using custom configurations would prevent Stackdriver from being
+  enabled.
+
+- **Fixed** an issue where long-lived connections(TCP bytes and gRPC) could result in a proxy memory leak.
```
Comment (windsonsea) [inline content/en/news/releases/1.18.x/announcing-1.18.7/index.md:30]:
```suggestion
- **Fixed** an issue where long-lived connections (TCP bytes and gRPC) could result in a proxy memory leak.
```
Not sure the space is intentionally removed?

Comment (hanxiaop) [reply content/en/news/releases/1.18.x/announcing-1.18.7/index.md:30]: Thanks, looks like we should have this space in the middle.

**Comment by jacob-delgado** [comment]
We should also include the 1.18.7 EOL announcement with this, similar to https://github.com/istio/istio.io/pull/14059/files

and also update https://istio.io/latest/docs/releases/supported-releases/ too

File: content/en/news/releases/1.18.x/announcing-1.18.7/index.md [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+
+This release contains bug fixes to improve robustness.
+
+This release note describes what’s different between Istio 1.18.6 and 1.18.7.
+
+{{< relnote >}}
+
+## Changes
+
+- **Fixed** a bug where overlapping wildcard hosts in a `VirtualService` would produce incorrect routing configurations
```
Comment (ericvn) [inline content/en/news/releases/1.18.x/announcing-1.18.7/index.md:18]:
```suggestion
- **Fixed** a bug where overlapping wildcard hosts in a `VirtualService` produces  incorrect routing configurations
```

File: content/en/news/releases/1.18.x/announcing-1.18.7/index.md [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+
+- **Fixed** an issue where `istioctl proxy-config ecds` didn't display all `EcdsConfigDump`.
+
+- **Fixed** an issue where new endpoints may not be sent to proxies.
+  ([Issue #48373](https://github.com/istio/istio/issues/48373))
+
+- **Fixed** an issue where installing with Stackdriver and using custom configurations would prevent Stackdriver from being
+  enabled.
+
+- **Fixed** an issue where long-lived connections (TCP bytes and gRPC) could result in a proxy memory leak.
```
Comment (ericvn) [inline content/en/news/releases/1.18.x/announcing-1.18.7/index.md:30]:
```suggestion
- **Fixed** an issue where long-lived connections, TCP bytes and gRPC, could result in a proxy memory leak.
```

**Comment by hanxiaop** [comment]
> We should also include the 1.18.7 EOL announcement with this, similar to https://github.com/istio/istio.io/pull/14059/files
> 
> and also update https://istio.io/latest/docs/releases/supported-releases/ too

Great, thanks for the info!

File: content/en/news/releases/1.18.x/announcing-1.18.7/index.md [hunk 4]
```diff
@@ -0,0 +1,30 @@
...
+
+This release contains bug fixes to improve robustness.
+
+This release note describes what’s different between Istio 1.18.6 and 1.18.7.
+
+{{< relnote >}}
+
+## Changes
+
+- **Fixed** a bug where overlapping wildcard hosts in a `VirtualService` produces  incorrect routing configurations
```
Comment (ericvn) [inline content/en/news/releases/1.18.x/announcing-1.18.7/index.md:18]:
```suggestion
- **Fixed** a bug where overlapping wildcard hosts in a `VirtualService` produces incorrect routing configurations
```

**Review by ericvn — APPROVED** [review]

**Pull request merged**